lib/process-services-cloud/src/lib/task/task-list/components/service-task-list-cloud.component.ts

```diff
@@ -55,9 +55,9 @@ export class ServiceTaskListCloudComponent extends BaseTaskListCloudComponent {
 
             combineLatest([
                 this.serviceTaskListCloudService.getServiceTaskByRequest(this.requestNode),
-                this.isColumnSchemaCreated$
+                this.columnsSchemaSubject$
             ]).pipe(
-                take(1)
+                takeUntil(this.onDestroy$)
```

Comment:
I've seen this change a lot, doesn't this mean that we will calling a lot of this calls? i think take(1) gave us the insurance to have been called once. Are we sure we are not keep on calling for everything that happens on the DL?